config/horizon.json

```diff
@@ -61,9 +61,6 @@
 		"jetpack/features-section/simple": true,
 		"jetpack/features-section/atomic": true,
 		"jetpack/features-section/jetpack": true,
```

Comment:
It seems we should have had `"jetpack/scan-product": true` here as well. It's activated in all other environments.